pkg/rancher-desktop/assets/scripts/moproxy.initd

```diff
@@ -0,0 +1,75 @@
+#!/sbin/openrc-run
+
+name=moproxy
+
+description="A transparent TCP to SOCKSv5/HTTP proxy."
+
+extra_started_commands="enable disable reload"
+description_enable="Start redirecting the network traffic to the HTTP proxy."
+description_disable="Stop redirecting the network traffic to the HTTP proxy."
+description_reload="Reload the proxy list."
+
```

Comment:
Should `moproxy.initd` have any dependencies, since we are relying on iptables?
e.g:
```
depend() {
    after iptables ip6tables
}
```